ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -255,6 +255,46 @@ tests:
     test:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-f60-compliance
+  cron: 49 7 27 2,4,6,8,10,12 *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
```

Comment:
You also need to add `FILTERS_ADDITIONAL: ""`. Otherwise the NonPreRelease has no chance to run.